src/main/java/teammates/storage/api/FeedbackQuestionsDb.java

```diff
@@ -52,6 +54,22 @@ public FeedbackQuestionAttributes getFeedbackQuestion(
         return makeAttributesOrNull(getFeedbackQuestionEntity(feedbackSessionName, courseId, questionNumber));
     }
 
+    /**
+     * Gets feedback question count grouped by giver types of a session.
+     */
+    public Map<FeedbackParticipantType, Long> getFeedbackQuestionGiverTypeCountForSession(
+            String feedbackSessionName, String courseId) {
+        assert feedbackSessionName != null;
+        assert courseId != null;
+
+        return load()
+                .filter("feedbackSessionName =", feedbackSessionName)
+                .filter("courseId =", courseId)
+                .list()
+                .stream()
+                .collect(Collectors.groupingBy(FeedbackQuestion::getGiverType, Collectors.counting()));
```

Comment:
I know this is practical, but this construct returns `Long`, which we really don't need, and as you can see you have to either add `L` suffix or wrap `Long.value` in many, many places.

`Collectors.counting()` should be replaced by a similar method that returns `Integer`. There is one.